README.md

```diff
@@ -27,14 +25,18 @@ Built into this container are 4 input plugins to collect metrics from the NuoDB
 4.  `threads` - extends the [Telegraf ProcStat Input plugin](https://github.com/influxdata/telegraf/tree/master/plugins/inputs/procstat) with per-thread data.
 Collects host machine resource consumption statistics on a regular 10s interval.
 
+To setup NuoDB Insights visual monitoring which uses the NuoDB Collector, follow the instructions on the [NuoDB Insights](https://github.com/nuodb/nuodb-insights) github page.
+
+to setup NuoDB perfomrance metric collection using NuoDB Collector when not using NuoDB Insights, follow the instruction on this page
```

Comment:
typo in performance. I also think that you need a capital T and a dot at the end.

Reply:
Thanks for finding those.